src/test/convertion.test.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+const sheetUtils = require('../main/lib/ccd-spreadsheet-utils');
+
+describe('JsonHelper', () => {
+  const jsonWithNumbers = ()=>[
+    { 'LiveFrom': 42736, 'LiveTo': 42736, 'ID': 1, 'Name': 'name' },
+    { 'LiveFrom': 42736, 'LiveTo': 42736, 'ID': 1, 'Name': 'name' }
```

Comment:
Might be good to make values undefined in second row to make sure it does not brake when date is unspecified.